test/Common.FunctionalTests/ConfigurationChangeTests.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Server.IIS.FunctionalTests.Utilities;
+using Microsoft.AspNetCore.Server.IntegrationTesting;
+using Microsoft.AspNetCore.Testing.xunit;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests
+{
+    [Collection(PublishedSitesCollection.Name)]
+    public class ConfigurationChangeTests : IISFunctionalTestBase
+    {
+        private readonly PublishedSitesFixture _fixture;
+
+        public ConfigurationChangeTests(PublishedSitesFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [ConditionalFact]
+        public async Task ConfigurationChangeStopsInProcess()
```

Comment:
Add a test that modifies apphost.config?

Reply:
Feels like this tests IIS not ANCM.

Reply:
ANCM should reload on apphost.config changes too. Difference is global vs local configurations.

Reply:
https://github.com/aspnet/IISIntegration/issues/1222